.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,6 +5,13 @@ title: ''
 labels: ''
 assignees: ''
 ---
+
+## Have you read the "Debugging SDKs" doc yet?
+
+There are common setup gotchas that happen with Percy's SDKs, it would be worth reading
+the debugging document, which might already answer your question:
+https://docs.percy.io/docs/debugging-sdk
+
 <!--
```

Comment:
Should the new paragraph be included in this comment (just move this line to be above it)? That way it won't end up in the issue description when somebody inevitabley doesn't remove excess parts of the template when creating a new issue.

Reply:
I battled with wrapping it in a comment or not. I guess it doesn't matter, if it's already answered in the doc, we're going to reply with that anyways.